docs/content/dataset_standards.md

```diff
@@ -299,3 +299,26 @@ Statistical metrics are also computed as metadata for the individual datasets in
     | `mean` | `np.float64` | Mean value of the episode rewards.         |
     | `std`  | `np.float64` | Standard deviation of the episode rewards. |
     | `sum`  | `np.float64` | Total undiscounted return of the episode.  |
+
+## Observation and Action Spaces
+The Minari storage supports the following observation and action spaces:
```

Comment:
Change the beginning of  this to "The Minari Storage format...

Reply:
done

Reply:
done